### ISSUE TITLE
Tilt without Gforce

### ISSUE THREAD

**Issue by itayrosenml** [kriswiner/MPU9250#278]
Hi,

I'm trying to detect a slope in a device (mobility scooter to be exact). Using Pitch, I'm able to get exact degrees of rotation. However, the values are affected by the Gforce and therefore when the device is in motion (e.g. driving, accelerating, slowing down), so does the Pitch. 

Is it possible to get rotation degrees around Yaxis which is not affected by Gforce?

**Comment by kriswiner** [comment]
You could remove gravity from the acceleration and re-calculate quaternions
I suppose...or just use the linear accelerations to estimate pitch.

**Comment by itayrosenml** [comment]
Hi,

Unfortunately acceleration is measured with Gforce, so I guess I need to somehow measure it differently.

**Comment by kriswiner** [comment]
Yes, but you can estimate gravity and gravity-less acceleration by
converting the quaternions into a rotation matrix. The using the
gravity-less accelerations and gyro, you can estimate tilt. See here
<https://github.com/kriswiner/Ladybug/blob/master/MPU9250_MS5637_BasicAHRS2_Ladybug.ino>
.

Line 558-560 are gravity-less linear acceleration
